drivers/dp/swdp_ll_pin.h

```diff
@@ -6,6 +6,7 @@
 
 #include <zephyr/kernel.h>
 #include <zephyr/drivers/gpio.h>
+#include <hal/nrf_gpio.h>
```

Comment:
That should not be necessary, maybe `#include <soc.h>`, only using definition from MDK here.

Reply:
The change is still there :-).

Reply:
ah, it was in the wrong commit, fixed it now